modulefiles/modulefile.hera

```diff
@@ -0,0 +1,38 @@
+#%Module#####################################################
+## Module file for tracker
+#############################################################
+proc ModulesHelp { } {
+    puts stderr "Set environment veriables for compiling or running HAFS"
+}
+module-whatis " HAFS model prerequisites"
```

Comment:
@BijuThomas-NOAA, Please change "HAFS" into "tracker" as well. Same for the module files for other platforms. Thanks!